demo/init-container-failure-in-cluster.demo

```diff
@@ -0,0 +1,39 @@
+# Demo: inject fault to initContainer in an existing cluster such as minikube or kind
+
+kubectl get nodes
+
+# Prepare manifest files
+make manifests
+sed -i -e 's/\/kwok:v0.5.0-94-g9b555d3-dirty/registry.k8s.io\/kwok\/kwok:v0.5.1/g' artifacts/kwok.yaml
+clear
+
+# Deploy and install kwok
+kubectl apply -f artifacts/kwok.yaml
+sleep 5
```

Comment:
Different environments may not have the same Internet speeds or image caches, so I feel like I should use `kubectl wait` instead